src/codeu/chat/ServerMain.java

```diff
@@ -15,9 +15,6 @@
 
```

Comment:
nit: it looks like your IDE formatter changed the indent from 2 to 4 spaces; i would recommend reverting to 2 spaces because it is in line with the Google style guide and also keeps in line with the skeleton code.